lifechanger/src/app/header/header.component.css

```diff
@@ -0,0 +1,85 @@
+.header {
+  height: 6.2em;
+  display: flex;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 100;
+  & .header__nav {
+    display: flex;
+    justify-content: space-between;
+    position: fixed;
+    background-color: var(--white);
+    top: 2.6em;
+    left: 20.8em;
+    width: 76rem;
+    /* height: 59px; */
```

Comment:
Is it required comment ? If we talking about comments in code. It is good approach to use for documentation of some logic (example xml comments in C# - attached image) Any other comments that related to code is better not to left. From my personal experience at project, we don't use comments for hiding some part of logic. So, my suggestion - is to remove them, if you don't mind. But i open for discussion, if you have some points here 
![image](https://github.com/TEZv/Become_PRO_with_Angular/assets/34271977/28ade22f-97cc-4588-9d3b-94caf63abd59)

Reply:
Thank you, dear Mentor.
I deleted the unnecessary comments, please, check once again:)